core_v2/sources/domains.move

```diff
@@ -131,9 +131,14 @@ module aptos_names_v2::domains {
     /// the reverse lookup has been cleared (in which case |target_address|
     /// will be none)
     struct SetReverseLookupEvent has drop, store {
```

Comment:
nit: could we rename `next_` to `curr_` to represent that this is the current state after the txn is submitted. 

Also could we include prev_ and curr_ expiration_time_secs this event.